services/facade/package.json

```diff
@@ -8,6 +8,7 @@
     "posttest": "npm run lint && nsp check"
   },
   "dependencies": {
+    "bluebird": "latest",
```

Comment:
Please do not depend on `latest`. Always use `^x.y.z`, `"bluebird": "^3.5.0"` in this particular case.

If you depend on `latest` and a new major version of your dependency is released, your application is very likely to break, because a new major version typically has backwards-incompatible changes.